server/tests/test_manual_data_input.py

```diff
@@ -13,11 +13,15 @@ def test_predefined_create_from_scratch(vanilla_user_client):
     url = reverse(
         "manage_indicator_data", args=[ind.id, period.id, sex_cat.pk]
     )
-    with patch_rules(can_edit_indicator_data=True):
```

Comment:
Idk if we're testing this elsewhere, but we should test the case in which can_edit is False and can_view is true, so that the read-only case is covered.